tests/ha/test_cluster_shutdown_start.py

```diff
@@ -996,21 +988,20 @@ def test_cluster_restart_without_signal(self):
             "Step 3: Cluster restarted fine and all Pods online.")
 
         LOGGER.info("Step 4: Check DI for IOs run before restart.")
-        resp = self.ha_obj.perform_ios_ops(di_data=di_check_data, is_di=True)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipwrite=True)
         assert_utils.assert_true(resp[0], resp[1])
-        self.s3_clean.pop(list(io_resp[2].keys())[0])
         LOGGER.info("Step 4: Verified DI for IOs run before restart.")
 
         LOGGER.info("Step 5: Create new S3 account and perform IOs.")
-        io_resp = self.ha_obj.perform_ios_ops(prefix_data='TEST-29479-1', nusers=1)
-        assert_utils.assert_true(io_resp[0], io_resp[1])
-        di_check_data = (io_resp[1], io_resp[2])
-        self.s3_clean.update(io_resp[2])
-        resp = self.ha_obj.perform_ios_ops(di_data=di_check_data, is_di=True)
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-29479-1'
+        self.s3_clean.update(users)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True)
```

Comment:
Should we add skipcleanup=True at all places where delete is not expected?
For e.g. line 992, 1228, etc.
Because delete is creating issues.

Reply:
So for tests where we are doing all IOs, keeping deletes as well and for others removing deletes